### PR TITLE
Donate Command

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [x] I have read and agreed to the [code of conduct](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [GitHub projects](https://github.com/orgs/CascadeBot/projects).
 - [x] I have tested all of my changes.
 
## Added/Changed
Add donate command

### CODE REVIEW

File: src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        EmbedBuilder builder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+        List<Page> pages = new ArrayList<>();
+
+        builder.setTitle(context.i18n("commands.donate.embed_title"));
+        builder.setDescription(context.i18n("commands.donate.embed_description"));
+        builder.setFooter(context.i18n("commands.donate.embed_footer"), context.getSelfUser().getAvatarUrl());
+        builder.addField("Patreon", context.i18n("commands.donate.embed_patreon"), true);
+
+        pages.add(new PageObjects.EmbedPage(builder));
+        for (Map.Entry<String, Tier> tier : Tier.getTiers().entrySet()) {
```
Comment (weeryan17) [inline src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java]: Add a check for default and nitro tiers as those cannot be donated to get

**Comment by JamesHawkinss** [comment]
@binaryoverload get ur ass in here

File: src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            if (tier.getKey().equals("default") || tier.getKey().equals("nitro")) {
+                continue;
+            }
+            String title = tier.getKey();
+            pages.add(new PageObjects.EmbedPage(new EmbedBuilder()
+                    .setDescription((tier.getValue().getGuildTierString(context.getLocale(), null)))
+                    .setTitle(title.substring(0, 1).toUpperCase() + title.substring(1))
```
Comment (binaryoverload) [inline src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java]: We have a util for this, use it!

File: src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        context.getUIMessaging().sendPagedMessage(pages);
+    }
+
+    @Override
+    public String command() {
+        return "donate";
+    }
+}
```
Comment (weeryan17) [inline src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java]: Blank line before end of class

Comment (JamesHawkinss) [reply src/main/java/org/cascadebot/cascadebot/commands/core/DonateCommand.java]: more of the clarification

**Pull request merged**